woocommerce/woocommerce
# Fix flaky tests, failures from Blocks merge

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Further updates to e2e tests to address flakiness.  The improved regex should match more price combinations.

Closes #41696
Closes #41691
Closes #41734
Closes #41607
Closes #41729
Closes #41825



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure CI passes
2. Pull branch
3. `pnpm env:restart && pnpm test:e2e-pw`



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
